.github/workflows/tests.yml

```diff
@@ -41,12 +44,14 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
+        with:
+          submodules: recursive
```

Comment:
What about adding clone document with submodules? Or I update robolectric.github.io after it merged.

Reply:
I think we can update the docs closer to the 4.7 release, this feature is still experimental and the build process may change,